### DISCUSSION TITLE
How to add a word count / expected reading time on each page?

### DISCUSSION THREAD

**Discussion by SmilingWayne** [squidfunk/mkdocs-material#4999] [Q&A] [answered]
Hello~😃

I'm wondering if a "**word count**" pattern( which can count how many words(characters) the current page has) and the approximate "Expected Reading Time" can be added to each page, either under the title or at the bottom of the page, just like the following pic shows. 

<img width="449" alt="pics wanted" src="https://user-images.githubusercontent.com/69518108/217032965-5999d496-bb2e-4df0-9aac-ed31e7bfe996.png">

I don't know for sure whether the total words and characters can be counted in markdown file.

**Comment by facelessuser** [comment, marked as answer]
It could easily be counted in HTML, I think that would be easiest. You could target and filter tags you do and don't want to count words in. It would make sense as a MkDocs plugin that maybe post-processes the content using certain configurable filters.

**Reply by SmilingWayne** [reply]
Thank you for your answer!😃 I overrides the content html to count words.
